### PR TITLE
Draft blog post for TYC workshop

### PR DESCRIPTION
**Pull request by jennaswa**

### CODE REVIEW

**Review by micaela-matta — APPROVED** [review]

File: _posts/2024-03-29-tyc-workshop.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: MDAnalysis Workshop with the Thomas Young Centre and CCPBioSim
+---
+
+MDAnalysis, in collaboration with the [Thomas Young Centre](https://thomasyoungcentre.org/), the [JCMaxwell Node of CECAM](https://uk-jcmaxwell.cecam.org/), and [CCPBioSim](https://www.ccpbiosim.ac.uk/), will deliver a hybrid workshop on an **Introduction to Molecular Dynamics Trajectory Analysis using MDAnalysis**. The workshop will take place at the University College London (United Kingdom) and online May 10, 2024 from 09:00 - 17:00 GMT.
+
+Places are limited, so registration is required before the deadline, **April 9, 2024**. There is a £10 fee for in-person registration, which includes lunch (but not travel or accommodation). Due to high demand, the online spaces are already full. If you are unable to join us in person, we encourage you to check out the recording after the event on the MDAnalysis YouTube channel.
```
Comment (RMeli) [inline _posts/2024-03-29-tyc-workshop.md:8]: Link to YouTube channel?

Comment (jennaswa) [reply _posts/2024-03-29-tyc-workshop.md]: Had meant to do that - thanks for the catch :)

File: _posts/2024-03-30-tyc-workshop.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: MDAnalysis Workshop with the Thomas Young Centre and CCPBioSim
+---
+
+MDAnalysis, in collaboration with the [Thomas Young Centre](https://thomasyoungcentre.org/), the [JCMaxwell Node of CECAM](https://uk-jcmaxwell.cecam.org/), and [CCPBioSim](https://www.ccpbiosim.ac.uk/), will deliver a hybrid workshop on an **Introduction to Molecular Dynamics Trajectory Analysis using MDAnalysis**. The workshop will take place at the University College London (United Kingdom) and online May 10, 2024 from 09:00 - 17:00 GMT.
```
Comment (orbeckst) [inline _posts/2024-03-30-tyc-workshop.md:6]: Is there really an article in front of "University College London"?

**Review by micaela-matta — APPROVED** [review]

File: _posts/2024-03-30-tyc-workshop.md (hunk 2 above)
Comment (micaela-matta) [reply _posts/2024-03-30-tyc-workshop.md]: no 😉

Comment (jennaswa) [reply _posts/2024-03-30-tyc-workshop.md]: oops :upside_down:

**Pull request merged**